collections/hub/other_tests/bulk_transfers/bulk-invalid-dfsp-name-header-and-body.json

```diff
@@ -201,10 +201,129 @@
               },
               {
                 "id": 7,
-                "description": "payer callback - should be error 3100. request.headers['FSPIOP-Destination'] participant not found.",
+                "description": "payer callback - should be error 3203. request.headers['FSPIOP-Destination'] participant not found.",
                 "exec": [
-                  "expect(callback.body.errorInformation.errorCode).to.equal('3100')",
-                  "expect(callback.body.errorInformation.errorDescription).to.equal(`Generic validation error - Participant {$request.headers['FSPIOP-Destination']} not found`)"
+                  "expect(callback.body.errorInformation.errorCode).to.equal('3203')",
+                  "expect(callback.body.errorInformation.errorDescription).to.equal(`Payee FSP ID not found - Participant {$request.headers['FSPIOP-Destination']} not found`)"
+                ]
+              }
+            ]
+          },
+          "ignoreCallbacks": false,
+          "params": {
+            "name": "testingtoolkitdfsp",
+            "ID": ""
+          },
+          "path": "/bulkTransfers",
+          "scriptingEngine": "javascript",
+          "scripts": {
+            "preRequest": {
+              "exec": [
+                "const delay = 1000 * 3600 * 24",
```

Comment:
Can we parametrise this please, and revert to the above default if said parameter is not set?

I.e. something like `const delay = environment.DELAY_THING_PARAM || 1000 * 3600 * 24`

Reply:
I found an example --> https://github.com/mojaloop/testing-toolkit-test-cases/blob/4d6c44aad47da64a4e68d0e49c84929d05bb5edb/collections/hub/golden_path/settlement_cgs/newsetcgs.json#L1554

Reply:
done.